.github/workflows/versioning.yml

```diff
@@ -39,4 +39,4 @@ jobs:
         id: build_number
         uses: onyxmueller/build-tag-number@v1
         with:
-          token: ${{secrets.GITHUB_TOKEN}}
\ No newline at end of file
+          token: ${{secrets.GITHUB_TOKEN}}
```

Comment:
What is this whitespace change? Don't we add a new line at every file end nowdays?